antarest/study/business/load_management.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+from io import BytesIO
+from typing import cast
+
+import pandas as pd
+from starlette.responses import Response
+
+from antarest.study.business.model.load_model import LoadDTO
+from antarest.study.model import Study
+from antarest.study.storage.rawstudy.model.filesystem.matrix.input_series_matrix import InputSeriesMatrix
+from antarest.study.storage.storage_service import StudyStorageService
+
+LOAD_PATH = "input/load/series/load_{area_id}"
+
+
+class LoadManager:
+    def __init__(self, storage_service: StudyStorageService) -> None:
+        self.storage_service = storage_service
+
+    def get_load_matrix(self, study: Study, area_id: str) -> Response:
```

Comment:
This "business" layer should be independent from web considerations, it should return a dataframe instead.

Formatting to arrow should be done in the web layer.

Reply:
Done